### PR TITLE
Project Plan- 4.28

### PR DESCRIPTION
**Pull request by elsazac**
Created Project plan for 4.28

@lshanmug

### CODE REVIEW

File: development/plans/eclipse_project_plan_4_28.xml [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+
+<p:compatibility_with_previous_releases>
+<div>
+
+<h3>Compatibility of Release 4.28 with 4.27</h3>
+<p>Eclipse 4.28 will be compatible with Eclipse 4.27</p>
+
+<p><strong>API Contract Compatibility:</strong> Eclipse SDK 4.28 will be upwards
+  contract-compatible with Eclipse SDK 4.27 except in those areas noted in the
+  <a href="https://www.eclipse.org/eclipse/development/porting/eclipse_4_27_porting_guide.html" target="_top">
```
Comment (lshanmug) [inline development/plans/eclipse_project_plan_4_28.xml:331]: Links should also be updated to point to 4_28 for example eclipse_4_28_porting_guide.html (though the page is not available yet). Same for the other 4_27 links.

**Comment by lshanmug** [comment]
Thanks @elsazac

**Comment by jarthana** [comment]
@lshanmug, @elsazac From 4.28 on, the minimum supported Java became 17. We should update the project plan to reflect this.

**Comment by jarthana** [comment]
@elsazac I also found the links to the freeze plan and porting guide broken. Looks like either the pages don't exist or we are using a wrong URL. We should fix them as well.

**Pull request merged**